doc/subsystems/logging/logger.rst

```diff
@@ -229,12 +230,11 @@ be used to change maximal severity level for given module. Module is identified
 by source ID and domain ID. Source ID can be retrieved if source name is known
 by iterating through all registered sources.
 
-If logger is processed from a thread then it is possible to enable a feature
+It is recommended to allow idle thread handle log processing by enabling
```

Comment:
A bit less complicated wording would be:

    You should allow idle thread handle log processing by enabling
    :option:`CONFIG_LOG_PROCESS_IN_IDLE``.  If you're using a custom
    thread to handle log processing, you can use
    :option:`CONFIG_LOG_PROCESS_TRIGGER_THRESHOLD` to wake up
    the processing thread when a certain number of log messages are buffered.